9.5/root/usr/share/container-scripts/postgresql/common.sh

```diff
@@ -172,19 +199,52 @@ function create_users() {
   fi
 }
 
+create_user_if_not_exists() {
+    local username="$1"
+    psql <<EOF
+DO
+\$body$
+BEGIN
+    IF NOT EXISTS (
+        SELECT * FROM pg_catalog.pg_user
+        WHERE usename = '${username}')
+    THEN
+        CREATE USER "${username}" LOGIN;
+    END IF;
+END
+\$body$
+EOF
+}
+
+function set_password() {
+    local user="$1" password="$2"
+    psql --command "ALTER USER \"${user}\" WITH ENCRYPTED PASSWORD '${password}';"
```

Comment:
There is `psql --set` option which should guard against (also not intentional) sql injection.